specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-06-01/domainservices.json

```diff
@@ -311,6 +311,242 @@
+          },
+          "201": {
+            "description": "HTTP 201 (Created) if the operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/ReplicaSet"
+            }
+          },
+          "202": {
+            "description": "HTTP 202 (Accepted) if the operation was successfully started and will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/ReplicaSet"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create ReplicaSet": {
+            "$ref": "./examples/CreateReplicaSet.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "get": {
+        "tags": [
+          "ReplicaSets"
+        ],
+        "operationId": "ReplicaSets_Get",
+        "summary": "Get ReplicaSet (GET Resources)",
+        "description": "The Get ReplicaSet operation retrieves a json representation of the ReplicaSet.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DomainServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ReplicaSetNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "HTTP 200 (OK) if the operation was successful.",
+            "schema": {
+              "$ref": "#/definitions/ReplicaSet"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get ReplicaSet": {
+            "$ref": "./examples/GetReplicaSet.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ReplicaSets"
+        ],
+        "operationId": "ReplicaSets_Delete",
+        "summary": "Delete ReplicaSet (DELETE Resource)",
+        "description": "The Delete ReplicaSet operation deletes an existing ReplicaSet.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DomainServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ReplicaSetNameParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "HTTP 202 (Accepted) if the operation was successfully started and will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/ReplicaSet"
+            }
+          },
+          "204": {
+            "description": "HTTP 204 (Not Content) should be used if the resource does not exist and the request is well formed."
```

Comment:
>HTTP 204 (Not Content) should be used if the resource does not exist and the request is well formed. [](start = 28, length = 100)

I think this description should say when the service returns 204. Perhaps something like: "HTTP 204 (No Content) if the resource does not exist."

Reply:
changed.